### DISCUSSION TITLE
Flacit.com - Music not recognizing

### DISCUSSION THREAD

**Discussion by Didd4Me** [aclap-dev/video-downloadhelper#524] [Help and Q&A] [answered]
### Your question

Hi,
a while ago I could download the music from allflac.com. Now VDH doesn't recognize any music anymore. Could somebody help me to get the music again?
I tried the stable and the beta version. Both don't work anymore.

### Addon & CoApp version and details

version: 8.2.2.8
channel: stable
build date: 2024-03-18
build options: linuxlic: false, noyt: false, browser: firefox
lang: de
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:126.0) Gecko/20100101 Firefox/126.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\didd",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by paulrouget** [comment]
Correct. I broke that sorry 😞  I'll work on a fix soon!

Thank you for the bug report.

Note to self: we ignore `content-disposition` header in extension matching.

**Reply by Didd4Me** [reply]
Any solution in sight? Is there anything I could do to help?

**Comment by Didd4Me** [comment, marked as answer]
I just switched to an older version.

**Comment by paulrouget** [comment]
This has been fixed in version 9.1 which is schedule to be released soon (early next month).
